Clover/app/src/main/java/org/floens/chan/core/presenter/ThreadPresenter.java

```diff
@@ -55,8 +55,8 @@
 import static org.floens.chan.utils.AndroidUtils.getString;
 
 public class ThreadPresenter implements ChanLoader.ChanLoaderCallback, PostAdapter.PostAdapterCallback, PostCellInterface.PostCellCallback, ThreadStatusCell.Callback, ThreadListLayout.ThreadListLayoutPresenterCallback {
-    private static final int POST_OPTION_QUOTE = 0;
-    private static final int POST_OPTION_QUOTE_TEXT = 1;
+    public static final int POST_OPTION_QUOTE = 0;
+    public static final int POST_OPTION_QUOTE_TEXT = 1;
```

Comment:
not sure if I should have made them all public to be consistent, so I only changed what I needed